<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>669</width>
     <height>120</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>演示</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">margin:right;</string>
         </property>
         <property name="text">
          <string>绘制：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>离散点</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>凸包</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>凸包切分</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delaunay三角形</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">margin:right;</string>
         </property>
         <property name="text">
          <string>点数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="colorBtn">
       <property name="text">
        <string>生成色彩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="genBtn">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">
margin:right;

font: 75 16pt &quot;Adobe Arabic&quot;;
font-color:red;</string>
       </property>
       <property name="text">
        <string>Author: WK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>图片处理</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="loadBtn">
         <property name="text">
          <string>导入图片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oriBtn">
         <property name="text">
          <string>原图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="triBtn">
         <property name="text">
          <string>三角化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">margin:right;

font: 75 16pt &quot;Adobe Arabic&quot;;
font-color:red;</string>
         </property>
         <property name="text">
          <string>Author: WK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
